packages/shopify-codemod/test/transforms/existential-assignment-to-if-statement.test.js

```diff
@@ -0,0 +1,8 @@
+import 'test-helper';
+import existentialAssignmentToIfStatement from 'existential-assignment-to-if-statement';
+
+describe('existentialAssignmentToIfStatement', () => {
+  it('only simplifies if-else statements that have their test, consequent, and alternate all match the coffeescript ?= decaf output', () => {
```

Comment:
A very long test name (especially with commas) is typically a smell.  A separate `ignores mismatched identifiers` sanity check would be helpful.